### PR TITLE
[yourporn] fix extraction and rename to sxyprn

### PR DESCRIPTION
**Pull request by polokomo**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/ytdl-org/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/ytdl-org/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/ytdl-org/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Bug fix
- [ ] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Fixes the extractor of https://sxyprn.com/ and renames it to SxyPrn. Support for https://yourporn.sex/ is droped, since that's a different website now.

### CODE REVIEW

File: youtube_dl/extractor/sxyprn.py [hunk 1]
```diff
@@ -7,10 +7,10 @@
 )
 
 
-class YourPornIE(InfoExtractor):
```
Comment (dstftw) [inline youtube_dl/extractor/sxyprn.py]: Do not touch extractor name.

File: youtube_dl/extractor/sxyprn.py [hunk 2]
```diff
@@ -33,11 +33,19 @@ def _real_extract(self, url):
 
         webpage = self._download_webpage(url, video_id)
 
-        video_url = urljoin(url, self._parse_json(
+        parts = self._parse_json(
             self._search_regex(
                 r'data-vnfo=(["\'])(?P<data>{.+?})\1', webpage, 'data info',
                 group='data'),
-            video_id)[video_id]).replace('/cdn/', '/cdn5/')
+            video_id)[video_id].split("/")
```
Comment (dstftw) [inline youtube_dl/extractor/sxyprn.py]: Single quotes.

File: youtube_dl/extractor/sxyprn.py [hunk 3]
```diff
@@ -33,11 +33,19 @@ def _real_extract(self, url):
...
+
+        num = 0
+        for c in parts[6]+parts[7]:
+            if c.isnumeric():
+                num += int(c)
+        parts[5] = str(int(parts[5])-num)
```
Comment (dstftw) [inline youtube_dl/extractor/sxyprn.py]: compat_str.

**Comment by v-delta** [comment]
Duplicate of #22255

**Pull request closed without merging**